src/main/resources/static/api.yaml

```diff
@@ -855,3 +872,34 @@ components:
             attribute: true
           format: <user-id>@<ncg-id>
           example: batman@ncg
+
+    LinkedCareContexts:
+      type: object
+      properties:
+        links:
+          type: array
+          items:
+            $ref: '#/components/schemas/links'
+    links:
+      type: object
+      properties:
+        hip:
+          type: object
+          properties:
+            id:
+              type: string
+            name:
+              type: string
+        careContexts:
+          type: array
+          items:
+            $ref: '#/components/schemas/MaskedCareContexts'
```

Comment:
Does this need to be masked anymore? Once the linkage is done, the masked references are revealed, right?